gerrit/ref-update

```diff
@@ -30,12 +30,13 @@ import sys
 import urllib.request
 
 # Edit this variable with the URL of the CLA manager instance
-CLA_MANAGER_URL = "https://cla-manager.example/check-cla"
+CLA_MANAGER_URL = "https://cla-manager/gerritWebhook"
```

Comment:
I was thinking of renaming this to `gerritEndpoint`, as it's not really a webhook.
```suggestion
CLA_MANAGER_URL = "https://cla-manager/gerritEndpoint"
```